src/build/webpack.config.js

```diff
@@ -185,6 +185,7 @@ function composeMaterialVersion(material) {
     commitHash = childProcess
       .execSync("git -c safe.directory=* rev-parse --short HEAD", {
         cwd: material,
+        env: { LANG: "C" } /* ensure language of output to catch error */,
```

Comment:
Could you explain a bit more why this is useful?

Reply:
Git is using the locale of the system and the error message can be in different language than the one tested in the `catch`.
Here it's forced to no language so english and the `catch` will work whatever the system's locale.